src/app/core/header/navbar/login-dialog/login-dialog.component.html

```diff
@@ -1,19 +1,6 @@
-<mat-dialog-content>
-  <h1 mat-dialog-title>
-    Login
-  </h1>
+<div class="dialog-wrap">
 
-  <mat-form-field>
-    <input placeholder="email" matInput [(ngModel)]="data.email">
-  </mat-form-field><br>
-  <mat-form-field>
-    <input placeholder="password" matInput [(ngModel)]="data.password">
-  </mat-form-field>
+  <dialog-login-form *ngIf="!loginDialogService.authorization"></dialog-login-form>
```

Comment:
do not access service method in the template. Wrap it in the method of the component instead.